packages/office-ui-fabric-react/src/components/SpinButton/examples/SpinButton.Controlled.Example.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import { SpinButton } from 'office-ui-fabric-react/lib/SpinButton';
+
+export interface ISpinButtonControlledExampleState {
+  value: string;
+}
+
+// tslint:disable:jsx-no-lambda
+export class SpinButtonControlledExample extends React.Component<{}, ISpinButtonControlledExampleState> {
+  public constructor(props: {}) {
+    super(props);
+
+    this.state = {
+      value: '37'
+    };
+  }
+
+  public render(): JSX.Element {
+    return (
+      <div style={{ width: '400px' }}>
+        <SpinButton
+          label="SpinButton as controlled component:"
+          value={this.state.value}
+          onValidValueUpdated={value => this.setState({ value })}
+        />
+        <br />
+        <SpinButton
+          label="SpinButton synchronous the value:will"
```

Comment:
>value:will [](start = 44, length = 10)

What does `value:will` mean?

Reply:
Sorry, that is a typo. I will remove the `will`.